src/components/CustomOverlayMap.tsx

```diff
@@ -56,7 +57,7 @@ export interface CustomOverlayMapProps {
  * Map에 CustomOverlay를 올릴 때 사용하는 컴포넌트 입니다.
  * `onCreate` 함수를 통해서 `CustomOverlay` 객체에 직접 접근 및 초기 설정 작업을 지정할 수 있습니다.
  */
-const CustomOverlayMap: React.FC<CustomOverlayMapProps> = ({
+const CustomOverlayMap: FCWithChildren<CustomOverlayMapProps> = ({
```

Comment:
매우 좋은 기여 감사합니다!

현재 `FCWithChildren` 이라는 type을 새롭게 만들어서 사용하고 있는데, 이 부분을 아래와 같이 `React` 에서 제공하는 `utility type` 으로 사용해주실 수 있을까요?
  
```suggestion
const CustomOverlayMap: React.FC<React.PropsWithChildren<CustomOverlayMapProps>> = ({
```

Reply:
수정했습니다!